packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -3,15 +3,17 @@ import { prompt } from 'inquirer'
 import * as colors from 'colors'
 
 const fs = require('fs')
```

Comment:
Use `fs-extra` for all file system operations and rename next line from `fsExtra` to `fs` - every method from `fs` is supported by `fs-extra`, so you can use it as a drop-in replacement.